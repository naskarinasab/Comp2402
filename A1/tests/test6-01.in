flavor
quiet
slimy
obscene
grandmother
rose
jam
rightful
mute
tidy
kitty
impulse
fool
mitten
hate
swanky
unsuitable
develop
capricious
wary
moan
found
spiky
