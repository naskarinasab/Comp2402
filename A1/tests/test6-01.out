jam
fool
hate
moan
mute
rose
tidy
wary
found
kitty
quiet
slimy
spiky
flavor
mitten
swanky
develop
impulse
obscene
rightful
capricious
unsuitable
grandmother
